src/Sylius/Bundle/TaxonomyBundle/Doctrine/ORM/TaxonRepository.php

```diff
@@ -116,6 +116,7 @@ public function findOneByName($name)
             ->leftJoin('o.translations', 'translation')
             ->where('translation.name = :name')
             ->setParameter('name', $name)
+            ->setMaxResults(1)
```

Comment:
Its true that it will, but only when using the find**One**ByName method. It will still display all the items when doing a findBy as it's not trying to find one. All this PR is doing is making the findOneByName method return the correct values that the default findOneByName would return (eg no Exception if multiple entries exist).